### PR TITLE
[website] Modify admin-api-topic.md document

### PR DESCRIPTION
**Pull request by yuruguo**
### Motivation
There are two parts need to adjust in [admin-api-topic.md](https://pulsar.apache.org/docs/en/next/admin-api-topics/)
1. List partitioned topics should use `list-partitioned-topics` rather than `list` in [manage-partitioned-topics/List](https://pulsar.apache.org/docs/en/next/admin-api-topics/#list-1). It involves `all versions` that support the command.
2. [Get backlog size](https://pulsar.apache.org/docs/en/next/admin-api-topics/#get-backlog-size) should not be placed in part [Manage partitioned topics](https://pulsar.apache.org/docs/en/next/admin-api-topics/#manage-partitioned-topics), because this command applies to  all topics included `no-partitioned-topics` and `partitioned-topics`, so it should be placed in [Manage topic resources](https://pulsar.apache.org/docs/en/next/admin-api-topics/#manage-topic-resources). It only involves `master` branch.

### Documentation
- [x] `doc`

### CODE REVIEW

**Comment by yuruguo** [comment]
@tuteng @Anonymitaet @Huanli-Meng PTAL, thx!

**Review by Huanli-Meng — APPROVED** [review]
LGTM from the document perspective. Please wait for technical review from the eng.

File: site2/docs/admin-api-topics.md
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:948]:
```suggestion
You can get the backlog size of a single partition topic or a non-partitioned topic with a given message ID (in bytes).
```
do you mean this?

File: site2/docs/admin-api-topics.md
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:1251]: @BewareMyPower can you review `line 1237 - line 1251` from the technical perspective? Thanks

File: site2/docs/admin-api-topics.md
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:946]: why move `get backlog size` here?

File: site2/docs/admin-api-topics.md
Comment (BewareMyPower) [reply site2/docs/admin-api-topics.md:1251]: It's okay.

**Review by BewareMyPower — APPROVED** [review]

File: site2/docs/admin-api-topics.md
Comment (yuruguo) [reply site2/docs/admin-api-topics.md]: Yes
Done, PTAL

**Pull request merged**
